cmd/terramate-lsp/main.go

```diff
@@ -96,14 +98,21 @@ type readWriter struct {
 func (s *readWriter) Close() error { return nil }
 
 func configureLogging(logLevel string, logFmt string, output io.Writer) {
-	zloglevel, err := zerolog.ParseLevel(logLevel)
-
-	if err != nil {
-		zloglevel = zerolog.FatalLevel
+	switch logLevel {
+	case "trace", "debug", "info", "warn", "error", "fatal":
```

Comment:
because ParseLevel accepts more levels than we document in the flags help (eg.: `panic`). I just added the same flags recognized by terramate here so we have a common set of log levels.